aggregator_core/src/task.rs

```diff
@@ -1363,64 +1346,4 @@ mod tests {
             ],
         );
     }
-
-    #[test]
-    fn reject_invalid_auth_tokens() {
```

Comment:
The new `janus_core::task::tests::reject_invalid_dap_auth_token` more narrowly tests that invalid `DapAuthToken`s are rejected.